corehq/sql_accessors/sql_templates/get_reverse_indexed_cases_3.sql

```diff
@@ -0,0 +1,33 @@
+DROP FUNCTION IF EXISTS get_reverse_indexed_cases(TEXT, TEXT[]);
```

Comment:
It looks like this changes the function quite a bit and uses the same function name. Can you change this to add a new function with a different name and then in a different PR dropping the old name? Doing it that way means we won't have interrmittent errors and it's possible to rollback

Reply:
Makes sense.  I'm guessing that's something we'd want to do for any SQL function changes that affect the parameters or return value.  Would be good to add to https://github.com/dimagi/commcare-hq/blob/master/corehq/form_processor/README.md#updating-a-sql-function .  I'll do that if no one else gets to it before I wrap up this PR.